web/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "description": "Frontend for the ereadingtool",
   "scripts": {
-    "start": "npm install && npm run build:dev && npm run dev",
+    "start": "npm install && npm run build && npm run",
```

Comment:
Can we keep these set to `dev`? We use these for local development to keep parcel running.

Reply:
Agreed